parser.js

```diff
@@ -199,6 +199,14 @@ Parser.propertiesParser = function(filename, content) {
   return PPARSER.parse(content, { namespaces: true, variables: true, sections: true });
 };
 
+Parser.numberParser = function(filename, content) {
+    return Number(content);
+}
+
+Parser.booleanParser = function(filename, content) {
+    return ('' + content === 'true');
+}
```

Comment:
Change both of these to:
```
Parser.booleanParser = (filename, content) => {
    return ('' + content === 'true');
}
```

Reply:
Given that the rest of this file uses `function()` instead of `() =>` I'd suggest leaving it as it is.